Tribler/community/tunnel/Socks5/server.py

```diff
@@ -219,25 +219,19 @@ def deny_request(self, request, reason):
         self.transport.write(response)
         self._logger.error("DENYING SOCKS5 request, reason: %s" % reason)
 
-    def on_udp_associate_request(self, connection, request):
-        if self.selection_strategy.has_options(self.hops):
-            # The DST.ADDR and DST.PORT fields contain the address and port that the client expects
-            # to use to send UDP datagrams on for the association.  The server MAY use this information
-            # to limit access to the association.
-            self._udp_socket = SocksUDPConnection(self, request.destination)
-            ip = self.transport.getHost().host
-            port = self._udp_socket.get_listen_port()
-
-            self._logger.info("Accepting UDP ASSOCIATE request to %s:%d", ip, port)
-
-            response = conversion.encode_reply(
-                0x05, conversion.REP_SUCCEEDED, 0x00, conversion.ADDRESS_TYPE_IPV4, ip, port)
-            self.transport.write(response)
-
-        else:
-            reason = "not enough circuits"
-            self.deny_request(request, reason)
-            self.close(reason)
+    def on_udp_associate_request(self, _, request):
```

Comment:
For clarity, I would prefer to not use the `_` in non-test code. If I ever want to extend/modify this method, I have to find out what this parameter is.